{"name": "test_allure", "status": "broken", "statusDetails": {"message": "TypeError: a bytes-like object is required, not 'int'", "trace": "self = <test_allure.TestAllure object at 0x0000027291293AC8>\n\n    @allure.step(title='woshiallure')\n    def test_allure(self):\n>       allure.attach(1,\"输入1\")\n\nscripts\\test_allure.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\_allure.py:164: in __call__\n    plugin_manager.hook.attach_data(body=body, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py:87: in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_pytest\\listener.py:215: in attach_data\n    self.allure_logger.attach_data(uuid4(), body, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\reporter.py:117: in attach_data\n    plugin_manager.hook.report_attached_data(body=body, file_name=file_name)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py:87: in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <allure_commons.logger.AllureFileLogger object at 0x0000027291155E10>, body = 1, file_name = 'e19e29c2-a0d5-4855-9b74-84efd78b1f2e-attachment.attach'\n\n    @hookimpl\n    def report_attached_data(self, body, file_name):\n        destination = os.path.join(self._report_dir, file_name)\n        with open(destination, 'wb') as attached_file:\n            if isinstance(body, text_type):\n                attached_file.write(body.encode('utf-8'))\n            else:\n>               attached_file.write(body)\nE               TypeError: a bytes-like object is required, not 'int'\n\n..\\..\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\logger.py:62: TypeError"}, "steps": [{"name": "woshiallure", "status": "broken", "statusDetails": {"message": "TypeError: a bytes-like object is required, not 'int'\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\allure_pratice\\scripts\\test_allure.py\", line 7, in test_allure\n    allure.attach(1,\"输入1\")\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 164, in __call__\n    plugin_manager.hook.attach_data(body=body, name=name, attachment_type=attachment_type, extension=extension)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\hooks.py\", line 286, in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py\", line 93, in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py\", line 87, in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 208, in _multicall\n    return outcome.get_result()\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 80, in get_result\n    raise ex[1].with_traceback(ex[2])\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_pytest\\listener.py\", line 215, in attach_data\n    self.allure_logger.attach_data(uuid4(), body, name=name, attachment_type=attachment_type, extension=extension)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\reporter.py\", line 117, in attach_data\n    plugin_manager.hook.report_attached_data(body=body, file_name=file_name)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\hooks.py\", line 286, in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py\", line 93, in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\manager.py\", line 87, in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 208, in _multicall\n    return outcome.get_result()\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 80, in get_result\n    raise ex[1].with_traceback(ex[2])\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\logger.py\", line 62, in report_attached_data\n    attached_file.write(body)\n"}, "attachments": [{"name": "输入1", "source": "e19e29c2-a0d5-4855-9b74-84efd78b1f2e-attachment.attach"}], "start": 1589125539603, "stop": 1589125539603}], "start": 1589125539602, "stop": 1589125539604, "uuid": "ed6d67a3-a959-4c6c-b294-f1b9580be11b", "historyId": "9e93c402c7371fb5dffa7e44cf24b3da", "testCaseId": "f4350440c035beeee20fd13fbd2b459e", "fullName": "scripts.test_allure.TestAllure#test_allure", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_allure"}, {"name": "subSuite", "value": "TestAllure"}, {"name": "host", "value": "WIN-DJ528GOHCBP"}, {"name": "thread", "value": "13412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_allure"}]}